DorisProject/src/app/pages/stock/stock.component.ts

```diff
@@ -1,6 +1,8 @@
 import {Component, OnInit} from '@angular/core';
 import {ProductService} from 'src/app/service/product.service';
 import {Router} from '@angular/router';
+import {ModalComponent} from '../modal/modal.component';
```

Comment:
Codacy has a fix for the issue: [Replace `ModalComponent` with `·ModalComponent·`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
import { ModalComponent } from '../modal/modal.component';
```